src/main/java/ro/unibuc/hello/service/AvionService.java

```diff
@@ -25,4 +25,9 @@ public InfoAvion getAvionInfoByNumber(String number) throws EntityNotFoundExcept
         }
         return new InfoAvion(counter.incrementAndGet(), String.format(avionTemplate,entity.number, entity.from, entity.to));
     }
+
+    public InfoAvion addAvion(Avion avion)  {
+        Avion entity = avionRepository.save(avion);
```

Comment:
Maybe we should treat a possible duplicate exception :two_women_holding_hands:  :ambulance:

Reply:
🗽